pkg/k8s/volumes/crud.go

```diff
@@ -47,29 +47,29 @@ func List(ctx context.Context, namespace, labels string, c kubernetes.Interface)
 }
 
 // CreateForDev deploys the volume claim for a given development container
-func CreateForDev(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset, devPath string) error {
+func CreateForDev(ctx context.Context, dev *model.Dev, c kubernetes.Interface, devPath string) error {
 	vClient := c.CoreV1().PersistentVolumeClaims(dev.Namespace)
-	pvc := translate(dev)
-	k8Volume, err := vClient.Get(ctx, pvc.Name, metav1.GetOptions{})
+	pvcFromDev := translate(dev)
```

Comment:
minor: wouldn't it be more appropriate `pvcForDev` instead `pvcFromDev` ? maybe even the function name already gives the necessary context and we can just use `pvc` what do you think?

Reply:
@AdrianPedriza I changed the name because I think it's the reason the bug got introduced, `pvc` is very generic and can lead to misuse. I think the best name would be `pvcInferredFromDev` since it's what we actually do, but `pvcForDev` also makes sense since we are going to use it as the PVC for the dev environment. 

I'll go with `pvcForDev` since it's what you felt was more intuitive and it's short and concise :+1: